osu.Desktop/OsuGameDesktop.cs

```diff
@@ -88,11 +83,15 @@ protected override void LoadComplete()
         {
             base.LoadComplete();
 
-            LoadComponentAsync(versionManager, Add);
+            LoadComponentAsync(versionManager = new VersionManager { Depth = int.MinValue });
+
             ScreenChanged += s =>
             {
-                if (!versionManager.IsPresent && s is Intro)
+                if (s is Intro && s.ChildScreen == null)
+                {
+                    Add(versionManager);
                     versionManager.State = Visibility.Visible;

```

Comment:
yep, one of these is redundant. i left too many in (found a pretty serious bug with VisibleContainer - see framework PR)